GVRf/Framework/src/org/gearvrf/GVRHybridObject.java

```diff
@@ -35,7 +30,7 @@
  * Descendant classes all have native (JNI) implementations; this base class
  * manages the native lifecycles.
  */
-public abstract class GVRHybridObject implements Closeable {
+public abstract class GVRHybridObject {
```

Comment:
If anyone wonders - using this interface triggers very annoying static analysis error in Eclipse. The tool thinks this entity is some kind of a stream (which it is not) and if it doesn't see the corresponding close complains. And close on this class is generally not supposed to be called.